src/main/kotlin/gg/octave/bot/commands/music/NowPlaying.kt

```diff
@@ -43,8 +43,15 @@ class NowPlaying : MusicCog {
                 true
             )
             addBlankField(true)
-            addField("Repeating", manager.scheduler.repeatOption.name.toLowerCase().capitalize(), true)
+
             addField("Volume", "${manager.player.volume}%", true)
+            addBlankField(true)
+            addBlankField(true)
```

Comment:
Remove the two addBlankField(true) and set inline parameter to false instead. (Format is broken on mobile.)
```suggestion
            addField("Volume", "${manager.player.volume}%", false)
```